src/mlpack/tests/main_tests/range_search_test.cpp

```diff
@@ -292,7 +298,7 @@ BOOST_AUTO_TEST_CASE(ModelCheck)
   CheckMatrices(neighbors, neighborsTemp);
   CheckMatrices(distances, distancetemp);
 
-  BOOST_REQUIRE_EQUAL(ModelToString(outputModel),
+  REQUIRE(ModelToString(outputModel) ==
```

Comment:
Hey @Aakash-kaushik, The indentation seems of in 302. Could you take a look at this. Generally we use 2 tabs or it matches the parent parenthesis. For more info you could take a look at style guidelines.

Reply:
These were this way in the [original file](https://github.com/mlpack/mlpack/blob/6ec3912a0864a65cfe18d4ecd8e7dee375822bb8/src/mlpack/tests/main_tests/range_search_test.cpp#L295) too, So i thought to leave them that way. if you say i will update the indentation.

Reply:
I think the bracket matched with the previous line but now it doesn't since the number of characters are different.

Reply:
okay i understand they are aligned with the parent parenthesis, sorry for overlooking it.

Reply:
I will quickly make all the requested changes.

Reply:
No worries at all. Thanks for the PR.